pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [tool.poetry]
-name = "acapy-vc-authn-oidc"
+name = "vc-authn-oidc"
```

Comment:
Is the name change necessary? Maybe related to https://github.com/openwallet-foundation/acapy-vc-authn-oidc/issues/701 ?

Reply:
Whoops no that was a merge mistake. Reversing. Thanks for catching